lib/cli/base.js

```diff
@@ -13,18 +13,24 @@ export default function(cli) {
 		"This option will disable resolution of node package dependencies.",
 			type: "string"
 		})
-		.option("verbose", {
-			describe: "Enable verbose logging.",
-			type: "boolean"
-		})
 		.option("loglevel", {
 			alias: "log-level",
-			describe: "Set the logging level (error|warn|info|verbose|silly).",
+			describe: "Set the logging level (silent|error|warn|info|perf|verbose|silly).",
 			default: "info",
 			type: "string"
 		})
-		.option("x-perf", {
-			describe: "Outputs performance measurements",
+		.option("verbose", {
+			describe: "Enable verbose logging.",
+			default: false,
+			type: "boolean"
+		})
+		.option("silent", {
+			describe: "Disable all log output.",
+			default: false,
+			type: "boolean"
+		})
+		.option("perf", {
```

Comment:
so `perf` is no longer experimental?

Reply:
Yes. The CLI option was only used in the `ui5 tree` command for some benchmarking. Now it also sets the corresponding log level, which makes sense I think.